### PR TITLE
43 tier 1 create admin componentpage

### PR DESCRIPTION
**Pull request by warrenchan13**
Finished admin user privileges (Create, Edit, Delete) with one minor bug

1. Not logged in and non-admin user view:
Screenshot 2023-01-16 at 10 57 20 PM

Screenshot 2023-01-16 at 10 58 00 PM

2. Admin only view:
Screenshot 2023-01-16 at 10 58 31 PM

Screenshot 2023-01-16 at 10 58 45 PM

Screenshot 2023-01-16 at 10 58 59 PM

### CODE REVIEW

File: client/app/AppRoutes.js [hunk 1]
```diff
@@ -28,14 +29,21 @@ const AppRoutes = () => {
 
   return (
     <div>
-      {isLoggedIn ? (
+      {isLoggedIn && isAdmin ? (
         <Routes>
```
Comment (robertdavidwest) [inline client/app/AppRoutes.js:33]: I think you need to pull. I added a route call "shop" in my last PR

File: client/features/singleProduct/EditSingleProduct.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+              value={productName}
+              onChange={(event) => setProductName(event.target.value)}
+              required
+            />
+          </FormControl>
+
+          <FormControl>
+            <InputLabel id="brand">Brand</InputLabel>
+            <Select
+              labelId="size"
```
Comment (jahedprince) [inline client/features/singleProduct/EditSingleProduct.js:93]: Edit "labelId" to brand

File: client/features/products/CreateProduct.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+              value={productName}
+              onChange={(event) => setProductName(event.target.value)}
+              required
+            />
+          </FormControl>
+
+          <FormControl>
+            <InputLabel id="brand">Brand</InputLabel>
+            <Select
+              labelId="size"
```
Comment (jahedprince) [inline client/features/products/CreateProduct.js:80]: edit labelId to brand

File: client/features/singleProduct/SingleProduct.js [hunk 4]
```diff
@@ -47,8 +51,11 @@ const SingleProduct = () => {
               <MainImage src={product.imageURL[selectedImage]} />
             </Grid>
             <Grid item sm={6}>
-              <Info product={product} />
+              <Info product={product} isAdmin={isAdmin} />
```
Comment (jahedprince) [inline client/features/singleProduct/SingleProduct.js:54]: remove isAdmin

**Review by robertdavidwest — APPROVED** [review]

**Pull request merged**